### ISSUE TITLE
Silent Hill 2 Remake FPSunlocked doesn't work.

### ISSUE THREAD

**Issue by NimboYT** [RoseTheFlower/UltrawideIndex#83]
Apparently the fix only unlocks the fps counter number, but the cutscenes look like they're still 30 fps.

**Comment by funkee69** [comment]
Cutscene still in 30 fps but the fps counter is unlocked

**Comment by Rancid1989** [comment]
Came to second this. I came here from a Steam post: https://steamcommunity.com/app/2124490/discussions/0/4849904828209657104/?ctp=5

As you can see from the comments, it's not working for people commenting either.

Let's hope we get a fix!

BTW, not sure if it is related or not, but i noticed the game launches 2 exes. I tried to limit FPS to 120 on both using nvidia control panel because my GPU went nuts on menu as you can hear an unreasonable Coil Whine i don't get on other games unless i fully uncap FPS. 

The FPS on menu now show to be capped but i can still hear the Coil Whine, so not sure if the counter are showing the real fps here, too.

**Comment by artemererer** [comment]
Same problem here

**Comment by Dark-talon** [comment]
the real issue seems to be that ANIMATIONS are locked to 30 fps still, i can see the fog and grass moving around in full frame rate during the second cutscene when you are talking to Angela.

**Comment by Rancid1989** [comment]
> the real issue seems to be that ANIMATIONS are locked to 30 fps still, i can see the fog and grass moving around in full frame rate during the second cutscene when you are talking to Angela.

There are a lot of animations hard capped at 30 FPS. Like curtains on the second save point house. This is obviously because this is a console port. There is no reason for them to do It on PC in 2024 lol

**Comment by RoseTheFlower** [comment]
From analyzing a 60+ FPS video of a cutscene frame by frame, there are cases of change with every frame, but as noted in the last comments, that's not true for every kind of animation. I'm looking further into it, but if it comes from the actual skeletal or other positions and not an additional engine cap, my hands would be tied. From there, it's up to each user to decide whether the cutscenes feel generally smoother or better with the unlocker in place.

**Comment by Rancid1989** [comment]
> From analyzing a 60+ FPS video of a cutscene frame by frame, there are cases of change with every frame, but as noted in the last comments, that's not true for every kind of animation. I'm looking further into it, but if it comes from the actual skeletal or other positions and not an additional engine cap, my hands would be tied. From there, it's up to each user to decide whether the cutscenes feel generally smoother or better with the unlocker in place.

I really don't understand how a game that is going to release in both platforms comes with so many animations hard-locked on 30fps, specially in such a good looking game. How are we not going to notice that? I'm in sure they were totally aware of this. It's a shame because besides this, it looks effing awesome. I really hope they or someone manages to fix this.

**Comment by OblivionIsAtHand** [comment]
Any progress on potentially fixing the hard-locked animations?

**Comment by BigPrimeNumbers** [comment]
Yes doesn't seem to work as expected, at least on the version at launch today.

**Comment by Doze-D** [comment]
The game needs optimization patches from the devs too, so I assume that within week or two, this mod should work just fine too.

**Comment by RoseTheFlower** [comment]
Lyall pointed me in the right direction as far as the animations and I've updated the fix to make them smoother.

**Comment by Doze-D** [comment]
> Lyall pointed me in the right direction as far as the animations and I've updated the fix to make them smoother.

Since it was mentioned on the description that it was only tested on three resolution, did you test out this with the usual ones? 
I'm 16:9 QHD player myself, so would wanna get this to work on that as well.

**Comment by BigPrimeNumbers** [comment]
> Lyall pointed me in the right direction as far as the animations and I've updated the fix to make them smoother.

Seems to work as expected now. Thanks so much for your work on these :)

**Comment by Rancid1989** [comment]
> Lyall pointed me in the right direction as far as the animations and I've updated the fix to make them smoother.

Amazing you guys managed to fix it! Thanks a lot!

Will we get a mod to also fix tha animations of other stuff capped at 30fps too? :D

**Comment by door665** [comment]
Worked perfectly to unlock the fps but this happens on the opening scene, looks funny at least 😁

https://share.icloud.com/photos/0d0mIzImNyjved9OdNfoYA97w

**Comment by RoseTheFlower** [comment]
> Worked perfectly to unlock the fps but this happens on the opening scene, looks funny at least 😁
> 
> https://share.icloud.com/photos/0d0mIzImNyjved9OdNfoYA97w

I don't remember that happening on my end. Seems to a game bug, but I'm not sure one way or another.

**Comment by agalarzad** [comment]
I also got that error in the initial cinematic, but it's not that the animation or James' teleportation is wrong, what happens is that for some reason the camera at that moment is not located on James' back, you can compare the video with the mod activated with the video with the original running at 30 fps

**Comment by door665** [comment]
> I also got that error in the initial cinematic, but it's not that the animation or James' teleportation is wrong, what happens is that for some reason the camera at that moment is not located on James' back, you can compare the video with the mod activated with the video with the original running at 30 fps

Yep, tested and only happens with the mod, it’s like that camera panning doesn’t exist with the mod

**Comment by Klemci** [comment]
Fixed with 1.1

**Comment by RoseTheFlower** [comment]
Good to know! I'm not quite sure what causes that camera / teleport / transition issue at the opening cutscene. Maybe the camera at that scene is somehow tied to the frame rate, so it goes off a bit. My changes are intentionally minimal as to not break anything, so I don't think the bug comes from me having done something wrong.

**Comment by Dark-talon** [comment]
> Can you fix animations outside of custcenes that are capped too?

oh yeah I did notice things like the cloth physics were only 30 fps

**Comment by Rancid1989** [comment]
Yeah, some animations like curtains on the room just before the first encounter are capped at 30 FPS.

Any chance we can get a fix for that too?

**Comment by TerryRed** [comment]
The only other issue I've found so far are flickering lights during the cut scene when you meet Eddie at the prison. (I'm at the end of the prison now).

**Comment by TerryRed** [comment]
After completing the game... most cut scenes play great with this fix. I did notice that when James bends down to put the vhs tape into the VCR the frame rate isn't smooth I think. The only other issues are some flickering lights during the scenes I mentioned above. There may be some others as I didn't use this fix until I was part way through the game. for the most part this works great. Just need those few issues sorted and it would make the game perfect (besides the transversal UE stutter issues, and controller haptic feedback cutting out).

**Comment by Klemci** [comment]
There has been no patch, dev will certainly change a lot of stuff, Ray Reconstruction...fixes

**Comment by Phurio93** [comment]
At the moment of playing VHS cutscene I got black screen with audio and subtitles working. I don't know if this is due to this mod or just a vanilla game bug, however I did't have any other problems with 1.1 version

**Comment by DitherMan420** [comment]
Hey I saw some reports that cutscenes are very stuttery with the new game patch and this mod. Can anyone confirm?

**Comment by DitherMan420** [comment]
Also seeing people saying the vanilla game cutscene fps cap is now 60 instead of 30 from the game patch.

**Comment by RoseTheFlower** [comment]
I have an old GPU, so maybe that's why I don't see any FG options. Combining that with the mod could be the issue.

**Comment by Klemci** [comment]
> I have an old GPU, so maybe that's why I don't see any FG options. Combining that with the mod could be the issue.

DLSS-Q**+FG** enabled, updated 1.04's game, mod presence looks to **work correctly** (i seen only first video when character after first fight breaks lthe windows and goes out btw).

7800x3d 4070ti

**Comment by PCLoadLetter2024** [comment]
Would it be possible to do this to the Indiana Jones 60fps cutscene cap too lol (if you're playing it)?

**Comment by RoseTheFlower** [comment]
> Would it be possible to do this to the Indiana Jones 60fps cutscene cap too lol (if you're playing it)?

It doesn't even run on my end, as my GPU does not support hardware ray tracing.

**Comment by PCLoadLetter2024** [comment]
> > Would it be possible to do this to the Indiana Jones 60fps cutscene cap too lol (if you're playing it)?
> 
> It doesn't even run on my end, as my GPU does not support hardware ray tracing.

There is a bypass. https://qiwi.gg/file/uAXL6613-TheGreatCirclePTBypass
